modules/serverless-send-events-kn.adoc

```diff
@@ -0,0 +1,48 @@
+[id="serverless-send-events-kn_{context}"]
+= Sending events
+
+You can use the `kn event send` command to send an event. The events can be sent either to publicly available addresses or to addressable resources inside a cluster, such as Kubernetes services, as well as Knative services, brokers, channels, and so on. The command uses the same builder-like interface as the `kn event build` command.
```

Comment:
Usually try to avoid phrases like "and so on." Are there actually more things here? Or would it be ok to say "Knative services, brokers, and channels." ?